src/App/component/AddRepositoryDialog.jsx

```diff
@@ -37,23 +54,21 @@ export default function AddRepositoryDialog({ open, reloadProjects, handleClose,
       checker.push(checkRepositoryURL(repositoryURL));
 
       Promise.all(checker)
-        .then((response) => {
+        .then(async (response) => {
           if (response.includes(false) === false) {
             let payload = {
               projectId,
               repositoryURL
             }
 
-            Axios.post(`http://localhost:9100/pvs-api/project/${projectId}/repository/${repoType}`, payload,
-              { headers: { "Authorization": `${jwtToken}` } })
-              .then(() => {
-                reloadProjects()
-                handleClose()
-              })
-              .catch((e) => {
-                alert(e.response?.status)
-                console.error(e)
-              })
+            try {
+              await Axios.post(`http://localhost:9100/pvs-api/project/${projectId}/repository/${repoType}`, payload, config)
```

Comment:
Suggest replace with `sendPVSBackendRequest` you just declared above.

Reply:
How to use `sendPVSBackendRequest` to send payload?

Reply:
For sending a request body by an `axios` instance, you can put it into the `requestConfig`'s `data` field.
please refer to [this line](https://github.com/axios/axios/blob/4fbf61dc751ad69c7df9977d6edf7cf7ee87a5fb/index.d.ts#L85).